pandas/tests/reshape/merge/test_join.py

```diff
@@ -803,3 +805,27 @@ def test_join_inner_multiindex_deterministic_order():
         index=MultiIndex.from_tuples([(2, 1, 4, 3)], names=("b", "a", "d", "c")),
     )
     tm.assert_frame_equal(result, expected)
+
```

Comment:
can you moe to test_cross_merge (same dir)

Reply:
If I understand you correctly, it is directly below the other cross tests